### ISSUE TITLE
Users can not scroll TableView with more than 10-12 items in iPad

### ISSUE THREAD

**Issue by omidgolparvar** [danielsaidi/Sheeeeeeeeet#36]
Hello
When we have lots of options (for example 15), in iPad (in my tests, iPad Pro 9.7 inch), `tableView.isScrollEnabled` is `false`, and users can NOT scroll `tableView` and select their choice. 

![simulator screen shot - ipad pro 9 7-inch - 2018-05-08 at 13 58 57](https://user-images.githubusercontent.com/5128342/39749474-03cf42d6-52c8-11e8-80bf-d5d6de51fc0f.png)

How can I fix this?

**Comment by danielsaidi** [comment]
Hi!

I'm not sure I follow. Do you mean that you have set `isScrollEnabled` to `false` for the popover table view or the main table view, or is it false by default? 

In the screenshot below, I have modified the `LinkActionSheet` initializer to add a large amounts of items to the sheet (`items: items + items + items + items`).

![simulator screen shot - ipad air 2 - 2018-05-08 at 15 19 41](https://user-images.githubusercontent.com/429927/39759639-7b20f3f6-52d3-11e8-83b5-4786d80ae895.png)

As you can see, this automatically enables scrolling for the popover table view. If you set `isScrollEnabled` to `false` for this table view, however, scrolling will of course be disabled.

One thing I've noticed, however, is that you present the popover from the cell. Since the cell spans over the entire screen, this will cause the popover to either present itself above or below the cell. I recommend presenting the popover for any of the subviews instead, since this will give the action sheet access to the full screen height.

**Comment by omidgolparvar** [comment]
In the code, `isScrollEnabled` is based on some calculations:

```
open var itemsViewRequiresScrolling: Bool {
    let required = requiredItemHeight
    let available = availableItemHeight
    return available < required
}

func positionItemsView(width: CGFloat) {
    ...
    itemsView.isScrollEnabled = itemsViewRequiresScrolling
}
```

I test your recommendation, and present the popover for one of the subviews (`titleLabel`). I checked code, and it seems that calculations are correct. But, as you noticed, the problem is, when I present popover for my view (`collectionViewCell`), popover present itself either above or below the cell; and therefore we have some space, that we can not use!

![simulator screen shot - ipad pro 9 7-inch - 2018-05-09 at 10 40 01](https://user-images.githubusercontent.com/5128342/39799127-5975d406-5378-11e8-9bdb-05afcb164b08.png)

In this image, the gray area is the space that popover did NOT use, but it's height was part of calculations result.

I think this is NOT a bug, but I don't know how to fix this! 😅😅

**Comment by danielsaidi** [comment]
Oh, you're absolutely correct! It's absolutely a bug. I've completely overlooked this when I adjusted how Sheeeeeeeeet calculates the available height. Not sure how to solve this, though.

**Comment by danielsaidi** [comment]
I have finally taken care of this bug. It will be solved in the upcoming `1.0` release.
https://github.com/danielsaidi/Sheeeeeeeeet/milestone/13
